src/openzeppelin/tests/access/test_accesscontrol.cairo

```diff
@@ -153,6 +158,12 @@ fn test_grant_role() {
     let mut state = setup();
     testing::set_caller_address(ADMIN());
     AccessControlImpl::grant_role(ref state, ROLE, AUTHORIZED());
+
+    let event = testing::pop_log::<RoleGranted>(ZERO()).unwrap();
```

Comment:
I like explicitly stating that we are querying the event from an address. In these cases is the ZERO address because we are not using the real call_contract_syscall (like dispatcher) so the context is not changed, but I think is important to have that explicit.

We may do something like this:

```
let event = get_event(ZERO(), RoleGranted);
```

But I like more something like:

```
let event: RoleGranted = testing::pop_log(ZERO()).unwrap();
```

Or just the version that we have tbh.

Reply:
Fair enough :) the version we have isn't difficult to write or read, I was just considering possible improvements

Reply:
i agree a small util to save us the unwrap and make it easier to read could add some value, but not really required

Reply:
Since we are now using helpers for asserting the events, I don't think adding a helper for the pop_log, to be inside another helper is that valuable.